CRM/Case/XMLProcessor/Report.php

```diff
@@ -986,4 +949,58 @@ public static function printCaseReport() {
     CRM_Utils_System::civiExit();
   }
 
+  /**
+   * Get other relationships for client.
+   *
+   * @param int $clientID
+   * @param int $caseID
+   * @param bool $isRedact
+   * @param CRM_Case_XMLProcessor_Report $report
+   * @param array $caseRelationships
+   *
+   * @return array
+   */
+  protected function getOtherRelationships(int $clientID, int $caseID, bool $isRedact, CRM_Case_XMLProcessor_Report $report, array $caseRelationships): array {
```

Comment:
I don't think caseID is used?

Reply:
Thanks for looking. Ah I forgot to remove it. I did the fix, which does use caseID, then went back and tried to make a smaller PR but forgot to take this out. `$rindex` too.